.travis.yml

```diff
@@ -19,19 +19,15 @@ gemfile:
   - gemfiles/Gemfile.rails-5.0.rb
 
 matrix:
-  allow_failures:
-    - rvm: jruby-19mode
-      gemfile: gemfiles/Gemfile.rails-4.2.rb
-      env: DB=postgres
-    - rvm: jruby-9.0.4.0
-      gemfile: gemfiles/Gemfile.rails-5.0.rb
   exclude:
-    - rvm: 2.0.0
-      gemfile: gemfiles/Gemfile.rails-5.0.rb
     - rvm: 2.1.0
       gemfile: gemfiles/Gemfile.rails-5.0.rb
     - rvm: 2.2.0
       gemfile: gemfiles/Gemfile.rails-5.0.rb
+    - rvm: 2.4.0
+      gemfile: gemfiles/Gemfile.rails-4.0.rb
+    - rvm: 2.4.0
+      gemfile: gemfiles/Gemfile.rails-4.1.rb
```

Comment:
https://travis-ci.org/Shuttlerock/friendly_id/jobs/232872362

```
TypeError: Cannot visit Integer
    /home/travis/build/Shuttlerock/friendly_id/gemfiles/vendor/bundle/ruby/2.4.0/gems/arel-5.0.1.20140414130214/lib/arel/visitors/visitor.rb:28:in `rescue in visit'
```

http://weblog.rubyonrails.org/2017/3/20/Rails-5-1-rc1/

> As per our maintenance policy, the release of Rails 5.1 will mean that bug fixes will only apply to 5.1.x, regular security issues to 5.1.x and 5.0.x, and severe security issues to 5.1.x, 5.0.x, and 4.2.x. This means 4.x and below will essentially be unsupported!

I think that we can skip `4.0` and `4.1` here too.